source/stylesheets/_landing.scss

```diff
@@ -125,68 +138,67 @@
   }
 }
 
-.actionable-sections {
-  padding: 0 $side-padding;
+.upcoming,
+.mailing-list {
+  background: $white;
+  padding: 3em $side-padding;
+}
+
+.upcoming {
+  border-bottom: 3px $blue-light solid;
+  margin: auto;
 
   @include media($medium-screen) {
-    @include clearfix;
-    @include outer-container;
-    position: relative;
+    @include display(flex);
+    @include padding(null 2.5em);
+    border: 3px $blue-light solid;
+    border-radius: $base-border-radius;
+    margin: 20vh auto;
+    max-width: 44em;
```

Comment:
You're using vh above does it makes sense to use vw? Or does that introduce issues?